server/src/server/env-config.js

```diff
@@ -25,6 +26,11 @@ const config = {
         level: process.env.NODE_ENV !== 'test' ? 'info' : 'error',
         prettyPrint: process.env.NODE_ENV !== 'test',
     },
+    aws: {
+        accessKeyId: process.env.AWS_ACCESS_KEY_ID,
+        secretAccessKey: process.env.AWS_SECRET_ACCESS_KEY,
```

Comment:
```suggestion
        accessKeyId: getRequiredEnv(process.env.AWS_ACCESS_KEY_ID),
        secretAccessKey: getRequiredEnv(process.env.AWS_SECRET_ACCESS_KEY),
```

Reply:
I might actually remove these from the config as these are especially sensitive,  they are not used anywhere as SES-API read them directly from env and we do not want to accidentally print them out.